docs/getting-started-guides/scratch.md

```diff
@@ -281,12 +281,12 @@ The kubeconfig file for the administrator can be created as follows:
    Guide), you will already have a `$HOME/.kube/config` file.
  - You need to add certs, keys, and the master IP to the kubeconfig file:
     - If using the firewall-only security option, set the apiserver this way:
-      - `kubectl config set-cluster $CLUSTER_NAME --server=http://$MASTER_IP --insecure-skip-tls-verify=true`
+      - `kubectl config set-cluster $CLUSTER_NAME-cluster --server=http://$MASTER_IP --insecure-skip-tls-verify=true`
```

Comment:
Just to be clear, you're adding the `-cluster` is for consistency with `kube-up.sh` generated files? It will work just as well the way it is now.